ui/src/app/pages/signed-in/access-renewal-notification.tsx

```diff
@@ -19,6 +19,8 @@ export const AccessRenewalNotificationMaybe = (
   const { profile } = useStore(profileStore);
   const daysRemaining = maybeDaysRemaining(profile, props.accessTier);
 
+  const moreThan14daysRemaining = daysRemaining > 13;
```

Comment:
`moreThan14daysRemaining` => `moreThan13daysRemaining` ?

Reply:
`>= 14` would capture what we we really want here